x/mongo/driver/operation.go

```diff
@@ -310,6 +310,7 @@ func (op Operation) Execute(ctx context.Context, scratch []byte) error {
 		}
 	}
 	batching := op.Batches.Valid()
+	retryWritesEnabled := op.RetryMode != nil && op.RetryMode.Enabled()
```

Comment:
Change variable name to `retryEnabled` or something similar because this doesn't only apply to writes.